### PR TITLE
Add store credit attributes to customer query

### PR DESCRIPTION
**Pull request by keharper**
## Purpose of this pull request

This pull request (PR) updates the `customer` query to include store credit attributes. 

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.3/graphql/reference/customer.html

whatsnew
Updated the [customer query](https://devdocs.magento.com/guides/v2.3/graphql/reference/customer.html
) to include store credit information.

### CODE REVIEW

File: guides/v2.3/graphql/reference/customer.md [hunk 1]
```diff
@@ -91,6 +94,42 @@ Attribute |  Data Type | Description
 --- | --- | ---
 `is_email_available` | Boolean | A value of `true` indicates the email address is available, and the customer can use the email address to create an account
 
+### Store credit attributes
+
+In {{site.data.var.ee}}, the merchant can assign store credit to customers. Magento maintains the history of all changes to the balance of store credit available to the customer. The customer must be logged in to access the store credit history and balance.
+
+Attribute |  Data Type | Description
+--- | --- | ---
+`store_credit` | [CustomerStoreCredit](#CustomerStoreCredit) | Contains the store credit information for the logged in customer
```
Comment (erikmarr) [inline guides/v2.3/graphql/reference/customer.md]:
```suggestion
`store_credit` | [CustomerStoreCredit](#CustomerStoreCredit) | Contains the store credit information for the logged-in customer
```

File: guides/v2.3/graphql/reference/customer.md [hunk 2]
```diff
@@ -91,6 +94,42 @@ Attribute |  Data Type | Description
...
+
+### CustomerStoreCredit attributes {#CustomerStoreCredit}
+
+The `store_credit` object contains store credit information, including the balance and history.
+
+Attribute |  Data Type | Description
+--- | --- | ---
+`current_balance` | Money | The current store credit balance
+`balance_history` | [`CustomerStoreCreditHistory`](#CustomerStoreCreditHistory) | Lists changes to the amount of store credit available to the customer. You can specify the following optional parameters to control paging in the output.<br/><br/>`pageSize` - An integer that specifies the maximum number of results to return at once. The default value is 20.<br/><br/>`currentPage` - An integer that Specifies which page of the results to return. The default value is 1
```
Comment (erikmarr) [inline guides/v2.3/graphql/reference/customer.md]:
```suggestion
`balance_history` | [`CustomerStoreCreditHistory`](#CustomerStoreCreditHistory) | Lists changes to the amount of store credit available to the customer. You can specify the following optional parameters to control paging in the output.<br/><br/>`pageSize` - An integer that specifies the maximum number of results to return at once. The default value is 20.<br/><br/>`currentPage` - An integer that specifies which page of the results to return. The default value is 1
```

File: guides/v2.3/graphql/reference/customer.md [hunk 3]
```diff
@@ -91,6 +94,42 @@ Attribute |  Data Type | Description
...
+The `CustomerStoreCreditHistory` object contains an array of store credit items and paging information.
+
+Attribute |  Data Type | Description
+--- | --- | ---
+`items` | [[`CustomerStoreCreditHistoryItem`](#CustomerStoreCreditHistoryItem)] | An array of products that match the specified search criteria
+`page_info` | SearchResultPageInfo | An object that includes the `page_size` and `current_page` values specified in the query
+`total_count` | Int | The number of items returned
+
+### CustomerStoreCreditHistoryItem attributes {#CustomerStoreCreditHistoryItem}
+
```
Comment (erikmarr) [inline guides/v2.3/graphql/reference/customer.md]: no lead in sentence?

File: guides/v2.3/graphql/reference/customer.md [hunk 4]
```diff
@@ -157,6 +196,110 @@ The following call returns information about the logged-in customer. Provide the
 }
 ```
 
+The following example returns the store credit history for the logged in user.
```
Comment (erikmarr) [inline guides/v2.3/graphql/reference/customer.md]:
```suggestion
The following example returns the store credit history for the logged-in user.
```

**Comment by keharper** [comment]
running tests

**Pull request merged**